Replicator/Replicator.cc

```diff
@@ -926,7 +942,9 @@ namespace litecore { namespace repl {
         Assert(json);
 
         MessageBuilder msg("setCheckpoint"_sl);
-        msg["collection"_sl] = coll;
+        if (_options->collectionAware) {
+            msg["collection"_sl] = coll;
```

Comment:
Need a note to switch to a constant here since this property will be widely used (I defined one in my PR)

Reply:
Also, in my next PR, I encapsulated it in a method of Worker, Worker::setMsgCollection(msg, collectionIndex). This method will decide whether to add the "collection" property based on _options

Reply:
What I meant is to eventually avoid the use of the "collection" magic string and instead have it available as a constexpr.

Reply:
Oh, yes. Can I take this advice in the next PR, which is also due this sprint.

Reply:
Done. adopted kCollectionProperty from Jim's PR